compiler/il/OMRILOps.cpp

```diff
@@ -291,3 +297,30 @@ OMR::ILOpCode::compareOpCode(TR::DataType dt,
    return TR::BadILOp;
    }
 
+/**
+ * \brief
+ *    Return a comparison type given the compare opcode.
+ *
+ * \parm op
+ *    The compare opcode.
+ *
+ * \return
+ *    The compareison type.
+ */
+TR_ComparisonTypes
+OMR::ILOpCode::getCompareType(TR::ILOpCodes op)
+   {
+   if (isStrictlyLessThanCmp(op))
+      return TR_cmpLT;
+   else if (isStrictlyGreaterThanCmp(op))
+      return TR_cmpGT;
+   else if (isLessCmp(op))
+      return TR_cmpLE;
+   else if (isGreaterCmp(op))
+      return TR_cmpGE;
+   else if (isEqualCmp(op))
+      return TR_cmpEQ;
+   else if (isNotEqualCmp(op))
+      return TR_cmpNE;
+   TR_ASSERT(false, "Expect comparison opcode");
```

Comment:
I think I can make the last `else if` an `else`. There is compare type other than `TR_cmpNE` that would reach there.